docs/backend/auth-twitter.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Twitter Authenication
+---
+
+# Twitter App and Authentication
+
+Dev.to allows you to authenicate using Twitter. In order to use this authenication method in local development you will need to setup a Twitter App and retrieves keys. Then you'll need to provide theses keys to the Rails application.
```

Comment:
authenicate -> authenticate
authenication -> authentication
retrieves keys -> retrieve the keys
theses -> these